### ISSUE TITLE
support anchor？

### ISSUE THREAD

**Issue by linwalker** [kriasoft/universal-router#95]
does it support anchor?
just like click on tag: <a href="#position1">jump</a>,
the page scroll to the position of the tag: <a name="positon1">position1</a>

**Comment by frenzzy** [comment]
Yes. This is the native browser behavior, you do not need to do anything special with the router. Demo: https://jsfiddle.net/frenzzy/e6bbfxk2/
